python/ray/_private/event/event_logger.py

```diff
@@ -157,3 +163,33 @@ def parse_event(event_str: str) -> Optional[Event]:
     except Exception:
         global_logger.exception(f"Failed to parse event: {event_str}")
         return None
+
+
+def filter_event_by_level(event: Event, filter_event_level: str) -> bool:
+    """Filter an event based on event level.
+
+    Args:
+        event: The event to filter.
+        filter_event_level: The event level string to filter by. Any events
+            that are lower than this level will be filtered.
+
+    Returns:
+        True if the event should be filtered, else False.
+    """
+
+    event_levels = {
```

Comment:
can you assert fail the non-existent level?

Reply:
`Event.Severity.Value(lvl)` should raise an error for the non-existent level.

Reply:
Hmm, are you sure you want to fail non-existent levels? This goes against `.proto` best practices & makes version compatibility harder.

Reply:
i guess the assertions was more on the supplied filter level - which could be not a valid level.